src/store/view-configs/common.ts

```diff
@@ -16,3 +23,45 @@ export const removeDataFromConfig =
       config.deleteSecondKey(dataID);
     }
   };
+
+export const ensureDefault = (key: string, records: Record<string, any>) => {
```

Comment:
This could be more generally useful in utils, e.g.

```typescript
export const ensureDefault = <T>(key: string, records: Record<string, T>, default: T) => {
  if (!(key in records)) records[key] = default;
})

ensureDefault(key, config, {})
```

This assumes that the type of config is known, and not assumed to be `any`.

Reply:
Makes sense ...

Reply:
Ok, good idea